components/brave_referrals/browser/brave_referrals_service.cc

```diff
@@ -63,6 +63,8 @@ const int kMaxReferralServerResponseSizeBytes = 1024 * 1024;
 // run.
 const char kDefaultPromoCode[] = "BRV001";
 
+using namespace brave_private_channel;  // NOLINT 
```

Comment:
`using namespace` is not allowed, see styleguide https://chromium.googlesource.com/chromium/src/+/master/styleguide/c++/c++.md

Reply:
- [x] removed `unsing namespace`